react/features/invite/components/InviteButton.web.js

```diff
@@ -185,13 +170,22 @@ class InviteButton extends Component {
             );
         }
 
+        const buttonOption = inviteItems[0];
+        const dropdownOptions = inviteItems.splice(1, inviteItems.length);
```

Comment:
This is working by coincidence right now, leveraging how splice essentially turns in push. I'll have to fix this before merging.

Reply:
Pushed a fix. The options get filtered first and then mapped.